### DISCUSSION TITLE
Output acceleration

### DISCUSSION THREAD

**Discussion by Zzzzsxb** [geodynamics/sw4#215] [General]
How to output peak surface acceleration

**Comment by houjun** [comment]
Currently there is no option to output the peak value, you can use the ssioutput command to output the entire time series of the surface (which can result in a large file) and manually calculate the peak values.

**Reply by Zzzzsxb** [reply]
If I use the 'image' command to output the velocity at each time step, and then calculate it manually, will this work?

**Comment by andersp** [comment]
Well, it depends on which source time function you use. If you use a function like the Gaussian that starts and ends with zero and the time-integral is positive, SW4 solves for the particle velocity. You can then save the max acceleration with the image modes 'hmaxdudt' and 'vmaxdudt', for the horizontal and vertical peak values. See UG section 11.5.3 for details.

**Reply by Zzzzsxb** [reply]
What is the command to save the maximum acceleration?There is no output command for acceleration.
